### DISCUSSION TITLE
Sunbird ED Mobile App - Support for Chromebooks ?

### DISCUSSION THREAD

**Discussion by parthlawate** [Sunbird-Ed/Community#254] [General Discussion]
Hello All, 

We have a use case where we need to install Sunbird App on Chromebook & showcase offline capability. Have we tested the Android app to work on Chromebooks ?  Technically its possible but is it tested ?

https://support.google.com/chromebook/answer/7021273?hl=en

Desktop is an exe .. so I assume it wont work on Chromebooks. 

Parth

cc @sharathkashyap

**Comment by swayangjit** [comment]
@parthlawate You can install apks on chromebook. It's not tested on Chromebook but it is tested on tablets so it should work.

**Reply by parthlawate** [reply]
Thanks @swayangjit We will test this & revert..
